financial-connections/api/financial-connections.api

```diff
@@ -382,6 +382,19 @@ public final class com/stripe/android/financialconnections/features/institutionp
 	public final fun getLambda-1$financial_connections_release ()Lkotlin/jvm/functions/Function2;
 }
 
+public final class com/stripe/android/financialconnections/features/linkaccountpicker/ComposableSingletons$LinkAccountPickerScreenKt {
```

Comment:
why are these generated?

Reply:
There's an open issue in the binary compatibility repository regarding this: https://github.com/Kotlin/binary-compatibility-validator/issues/74